nilearn/plotting/surf_plotting.py

```diff
@@ -11,7 +11,6 @@
 from matplotlib.colorbar import make_axes
 from matplotlib.cm import ScalarMappable, get_cmap
 from matplotlib.colors import Normalize, LinearSegmentedColormap
-from mpl_toolkits.mplot3d import Axes3D
```

Comment:
I think this is required for old matplotlib. Rather put a `#noqa` at the end of the line

Reply:
Done.